hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -64,10 +64,10 @@
           CORE_SITE_CONFIGURATION_FILE));
 
   @Evolving
-  public static final int APPLICATION_MAX_TAGS = 10;
+  public static final int APPLICATION_MAX_TAGS = 15;
 
   @Evolving
-  public static final int APPLICATION_MAX_TAG_LENGTH = 100;
+  public static final int APPLICATION_MAX_TAG_LENGTH = 200;
```

Comment:
Any rationale for 200?
Do you have a rough idea of the impact? memory, zk nodes?